ibis/udf/vectorized.py

```diff
@@ -85,9 +85,9 @@ def func(*args):
 
         op = self.func_type(
             func=func,
-            args=args,
+            func_args=args,
             input_type=self.input_type,
-            output_type=self.output_type,
+            return_type=self.output_type,
```

Comment:
I'd prefer moving the files from `ibis/udf` directory to `ibis/expr/udf` and define the API utilities there as well, similarly to [SearchedCaseBuilder](https://github.com/ibis-project/ibis/blob/master/ibis/expr/operations.py#L1575).

Reply:
That seems reasonable to me.

Reply:
I originally put it under `ibis` because this is meant to be an API namespace and I think most of stuff under ibis/expr are not API modules?

Reply:
Deferring it to a follow-up.